project/spotify/static/spotify/neighbors.js

```diff
@@ -0,0 +1,16 @@
+$(document).ready(function() {
+	
+	$.getJSON("/neighborsinfo",function(data) {
+		var users = data['user_list']
+		var neighbors = data['neighbors_list']
+		for (user_index in users) {
+			$("#neighbors").append(users[user_index] +"<ol>")
+			for (neighbor in neighbors[user_index]) {
```

Comment:
Creating global 'for' variable. Should be 'for (var neighbor ...'.<br>Identifier 'user_index' is not in camel case.<br>The body of a for in should be wrapped in an if statement to filter unwanted properties from the prototype.<br>'user_index' is not defined.